tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -0,0 +1,90 @@
+package com.techcourse.controller;
+
+import com.techcourse.db.InMemoryUserRepository;
+import com.techcourse.exception.UnauthorizedException;
+import com.techcourse.model.User;
+import jakarta.servlet.http.HttpSession;
+import java.util.List;
+import org.apache.catalina.controller.AbstractController;
+import org.apache.catalina.controller.Handler;
+import org.apache.catalina.controller.HttpEndpoint;
+import org.apache.catalina.session.SessionManager;
+import org.apache.coyote.http11.request.HttpMethod;
+import org.apache.coyote.http11.request.HttpRequest;
+import org.apache.coyote.http11.request.Queries;
+import org.apache.coyote.http11.response.HttpResponse;
+import org.apache.coyote.http11.response.HttpStatus;
+import org.apache.coyote.http11.response.ResponseCookie;
+import org.apache.coyote.http11.response.ResponseFile;
+
+public class LoginController extends AbstractController {
+
+    private static final SessionManager SESSION_MANAGER = new SessionManager();
+    private static final HttpEndpoint LOGIN_POST = new HttpEndpoint("/login", HttpMethod.POST);
+    private static final HttpEndpoint LOGIN_GET = new HttpEndpoint("/login", HttpMethod.GET);
+    private static final String LOGIN_PAGE = "/login.html";
+
+    public LoginController() {
+        List<Handler> handlers = List.of(
```

Comment:
클라이언트가 url 마다 객체를 만들어서 전달해주려면 불편하지 않을까요?

```suggestion
        List<Handler> handlers = List.of(
                Handler.ofPost("/login", this::doLoginPost),
                Handler.ofGet("/login", this::doLoginGet)
        );
```